src/main/java/org/cascadebot/cascadebot/commands/informational/UserInfoCommand.java

```diff
@@ -45,9 +39,9 @@ public void onCommand(Member sender, CommandContext context) {
             return;
         }
         Member member = context.getGuild().getMember(userForInfo);
-        
         String status = "";
         String statusName = "";
+        String footer = "Requested by " + sender.getUser().getAsTag();
```

Comment:
Remove this and instead get the builder with `MessagingObjects.getMessageTypeEmbedBuilder`